safe-global/safe-wallet-web
# Fix(Multichain): Show owner setup warning also when replacing owners and changing threshold [SW-150]

## What it solves

Resolves: https://www.notion.so/safe-global/Multichain-Safes-V1-change-ownership-91f4347bbf764d35b7fdfa54179d7972?pvs=4

## How this PR fixes it
- Shows the warning message whenever signer setup is changed.
- Also changes the blockaid warning to have the same icon as other warnings on the page.

## How to test it
- create a multichain safe
- change the signer setup of one of the chains.
- On the tx confirmation screen for adding, removing, replacing, or changing threshold, you should see the warning.

## Screenshots

Making icons consistant (before):
![image](https://github.com/user-attachments/assets/964c8b00-f571-42d1-ae75-0ecb1de7301b)

After:
![image](https://github.com/user-attachments/assets/f5819b9c-0cca-4b0e-aa7d-4558b35c2c4d)


## Checklist
